### ISSUE TITLE
Mail attachment are not correctly recognised by the email client

### ISSUE THREAD

**Issue by gnetsanet** [nextflow-io/nextflow#609]
As a workaround, I wrote the email content/body to a file and tried to attach that file with an 'inline' disposition but attaching multiple files does not work.

This is the case in versions 0.27.0 and 0.27.4

**Comment by pditommaso** [comment]
Not sure to understand. Can you provide a test case to replicate the problem?

**Comment by gnetsanet** [comment]
The following will attach the second attachment but not the first and does not include the body.

```
sendMail { 
          to 'myemail@mailsever.com'
          subject 'Test'
          body 'Email content or body'
          attach '/mnt/home/gnetsanet/novaseq_pipe/run_dec20.txt', fileName: 'run_dec20.txt'
          attach '/mnt/home/gnetsanet/novaseq_pipe/timeline.html', fileName: 'timeline.html'
        }

```
If the attach parameters are commented out, the email body makes it into the email.

```
sendMail { 
          to 'myemail@mailsever.com'
          subject 'Test'
          body 'Email content or body'
          //attach '/mnt/home/gnetsanet/novaseq_pipe/run_dec20.txt', fileName: 'run_dec20.txt'
          //attach '/mnt/home/gnetsanet/novaseq_pipe/timeline.html', fileName: 'timeline.html'
        }
```

**Comment by pditommaso** [comment]
Fascinating. Clearly a bug, thanks for reporting it.

**Comment by pditommaso** [comment]
It looks it's the ` fileName: <string>` option that messed up things.

**Comment by pditommaso** [comment]
I've upload a snapshot that should fix this issue. If you want to try it use the following command: 

```
NXF_VER=0.28.0-SNAPSHOT nextflow run .. etc
```

**Comment by gnetsanet** [comment]
Yes. Now, it works as expected. 

Thank you.

**Comment by pditommaso** [comment]
Fixed in version `0.27.5`.
